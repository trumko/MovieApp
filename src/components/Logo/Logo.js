import React from "react";
import { Link } from 'react-router-dom';

import './Logo.scss'

export default function Logo() {
  return (
    <div className="mr_logoConteiner">
      <Link to="/" className="mr_logo">netflixroulette</Link>
    </div>
  )
}
