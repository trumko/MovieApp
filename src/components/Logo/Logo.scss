@import "styles/variables.scss";

.mr_logoConteiner {
    margin: 10px 0 30px 0;
}

.mr_logo {
  color: $pink;
  font-family: sans-serif;
  font-weight: bold;
}

.mr_logo:hover {
  color: #fff;

}
