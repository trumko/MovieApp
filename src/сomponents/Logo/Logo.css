.mr_logo {
  color: #c13a3a;
  font-family: sans-serif;
  font-weight: bold;
}
