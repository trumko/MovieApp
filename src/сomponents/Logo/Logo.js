import React from "react";
import { Link } from 'react-router-dom';

import './Logo.css'

export default function Logo() {
  return (
    <Link to="/">
      <span className="mr_logo">netflixroulette</span>
    </Link>

  )
}
