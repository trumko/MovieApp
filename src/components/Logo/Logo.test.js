import React from 'react'
import { shallow } from 'enzyme'
import Logo from './Logo'

describe('Logo', () => {
  it('Logo should be rendered correctly', () => {
    const component = shallow(<Logo />);

    expect(component).toMatchSnapshot();
  });
});
