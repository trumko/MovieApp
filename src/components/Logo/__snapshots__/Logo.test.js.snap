// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo Logo should be rendered correctly 1`] = `
<div
  className="mr_logoConteiner"
>
  <Link
    className="mr_logo"
    to="/"
  >
    netflixroulette
  </Link>
</div>
`;
